### ISSUE TITLE
Add Alt Text for all images uploaded to WP Content 

### ISSUE THREAD

**Issue by ekeaton** [Trillium-Solutions/kern-theme#11]
All images that have been uploaded to posts and pages should have alt text. These can be updated in wp-admin, in the media library. 

You can filter to only show images

<img width="333" alt="Screen Shot 2021-09-30 at 1 50 55 PM" src="https://user-images.githubusercontent.com/43942870/135528276-f4cfdd8c-fa77-46fb-8808-226534c4b8dd.png">

And then when you click on an image to the right you'll see the input labeled "Alternative text". 
 
<img width="421" alt="Screen Shot 2021-09-30 at 1 52 50 PM" src="https://user-images.githubusercontent.com/43942870/135528498-38dd1866-f56b-40e7-b229-b2a188193671.png">

You can bill this work under 
(Kern County Roads Dept (Transit))

I'd say it shouldn't take more than an hour, so let me know if it does and if you have any other questions!

**Comment by danielleogden** [comment]
@MarkTrillium @ekeaton let's hold off on this until after Blindmast does their review. We don't have the budget at this point and we will charge them hourly for remediation of what Blindmast finds.

**Comment by MarkTrillium** [comment]
@lisaattrillium Are we still waiting on the Blindmast review?

**Comment by danielleogden** [comment]
@MarkTrillium  No, but the client has not approved the remediation work.

**Comment by ekeaton** [comment]
I went ahead and made this update while completing the additional accessibility updates
